### ISSUE TITLE
Consider changing Publish/Unpublish to Start/Stop

### ISSUE THREAD

**Issue by kcbabo** [syndesisio/syndesis#2896]
## This is a...
[X] Feature request
</code></pre>

## The problem
The terms publish and unpublish are pretty severe from an end user POV.  If we want to call the initial button to deploy "Publish", I'm OK with that, but I think the state changes available for deployed integrations should be

- Start (was publish)
- Stop (was unpublish)
- Delete

Thoughts?

## Expected behavior
Action names meet user expectations.  FWIW, I got this feedback from a user. :-)

**Comment by gashcrumb** [comment]
+1 from me

@TovaCohen @dongniwang @sjcox-rh ?

Wonder if that has a bigger impact on the docs, I think in the UI it's not such a big change to make.

**Comment by TovaCohen** [comment]
If a switch from Publish to Start, and from Unpublish to Stop makes the product easier and more intuitive to use, then yes, let's make the change. 

And let's make this change in concert with other changes to provide a consistent user experience. Specifically:

- Change the "Published" status to "Running".
- Change the "Unpublished" status to "Stopped".
- Change to Start and Stop on the kebab menu. 

And thanks, yes, Stan, it will take a couple of hours at least to update the doc because of the textual descriptions. It won't be a simple search and replace. But I'm certainly happy to do it.

**Comment by TovaCohen** [comment]
Suppose you do this:
1. Publish an integration for the first time. 
2. Unpublish that integration. 
3. Publish that same integration for the second time. 
Is the Syndesis behavior different for step 3 than it is for step 1?
If yes, then I think we need to distinguish between "Publish" for the first time and "Start" when it is not the first time.

**Comment by gashcrumb** [comment]
>Does this impact our status for publishing at all? I’m thinking specifically of the case where the user clicks start or stop on an Integration - does the UI show status updates for that?

Well, at the moment it's "Publishing" or "Unpublishing" for state changes, I'd change that to probably "Starting" and "Stopping", then we'll need to adjust whatever text we decide on for #2343 which is still WIP.

**Comment by kcbabo** [comment]
Let me rephrase this scenario using our new terms:

Suppose you do this:

   1. Publish an integration for the first time.
   2. Stop that integration.
   3. Start the same integration.

There is no unpublish any longer.  If someone edits an integration then
that’s a new version that they would publish.  If they want to get rid of
an integration completely they can delete the integration.

**Comment by TovaCohen** [comment]
Do I have this right:
**Publish** - the first time that you run the integration. The user will see the 5 stages of deploying an integration as implemented by `https://github.com/syndesisio/syndesis/issues/2343`. 
**Stop** - any time that you no longer run the integration
**Start** - running the integration any time after it has already been run once. How many stages and which stages would the user see in the  progress bar?

**Comment by kcbabo** [comment]
Start is running the integration any time after it has been stopped.

For the start stages, I think "Starting" is all we really need to show the user.  We need more stages for Publish because it takes so long.

**Comment by gashcrumb** [comment]
So this is coming along, wonder, this 'Draft' line here, I thought it best to keep "Publish" for that one, thoughts?

![capture](https://user-images.githubusercontent.com/351660/41916939-367081f2-7927-11e8-9896-6882355e6b4d.PNG)

**Comment by gashcrumb** [comment]
And just found dialogs and notifications that need updating when I clicked Start :-)

**Comment by dongniwang** [comment]
All changes sound good to me. Thank you! 

cc: @amysueg @sjcox-rh @michael-coker @seanforyou23

**Comment by gashcrumb** [comment]
@dongniwang phew!  'cause I just merged it :-)

**Comment by kcbabo** [comment]
Nice!
